# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/virt-viewer/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- virt-viewer-11.0/meson.build.vanilla	2024-04-28 17:12:19.450000000 +0200
+++ virt-viewer-11.0/meson.build	2024-04-28 17:12:29.014000000 +0200
@@ -573,8 +573,8 @@
 subdir('src')
 subdir('po')
 subdir('man')
-subdir('tests')
+#subdir('tests')
 if bash_completion_dep.found()
-  subdir('bash-completion')
+  #  subdir('bash-completion')
 endif
-subdir('data')
+#subdir('data')
